index.html

```diff
@@ -667,6 +667,61 @@ <h4>Use the new JSON-LD context when issuing new credentials</h4>
 
       </section>
 
+      <section>
+        <h3>Extending JWTs</h3>
+
+        <p>
+The Verifiable Credentials Data Model specifies a minimal set of 
+JWT <a>claim</a> names that are to be used to represent the
+<a>properties</a> of a <a>verifiable credential</a>
+and its <code>credentialSubject</code>. Implementers may wish to extend a
+<a>verifiable credential<a> with some
+<a>properties</a> that are new (e.g., <code>drivingLicenseNumber</code>,
+<code>mySpecialProperty</code> or that are already registered with IANA
+as JWT <a>claim/a> names (e.g., <code>given_name</code>.
+<code>phone_number_verified</code>. 
+      </p>
+      <p>
+As the Verifiable Credentials Data Model states, such extension properties are best
```

Comment:
I tried to put this line -- which is an HTML comment, so will not render in the human-friendly page -- into the document, so that others in the WG may notice and address the need --

```
<!-- Should link to the relevant section of the data model spec. -->
```

Reply:
I didn't realise this was what you were trying to do. So now I have inserted it